tests/page-objects/display/languages.po.js

```diff
@@ -1,5 +1,6 @@
 const helper = require('../../helper');
 const utils = require('../../utils');
+const { element } = require('protractor');
```

Comment:
Why is this needed now?

Reply:
Oh, I think my IDE autocompleted this and I failed to notice, so sorry!